crates/rustc_codegen_spirv/src/codegen_cx/mod.rs

```diff
@@ -831,6 +822,28 @@ impl<'tcx> HasTypingEnv<'tcx> for CodegenCx<'tcx> {
     }
 }
 
+impl<'tcx> CodegenCx<'tcx> {
+    // FIXME(eddyb) a bit odd that this isn't in `rustc_codegen_ssa`, instead of
+    // the pair of `get_fn` + `get_fn_addr`.
+    fn fn_addr_of(&self, span: Span, function: SpirvFunctionCursor) -> SpirvValue {
```

Comment:
Noticing during rebasing later work on top of this PR that this split doesn't actually work, and some other solution is required.